packages/table-react/src/TableRow.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { MouseEvent } from "react";
+import { TableRowData } from "./Table";
+import { TableData } from "./TableData";
+
+export type TableRowDataVariants = "anchor";
+
+export interface BaseTableRowData {
+    type: TableRowDataVariants;
+}
+
+export interface TableAnchorRowData extends BaseTableRowData {
+    rowData: string[];
+    type: "anchor";
+    href: string;
+    hrefLabel: string;
+    onRowClick?: (href: string) => void;
+}
+
+interface Props {
+    row: TableRowData;
+}
+
+export function TableRow(props: Props) {
+    const { row } = props;
+    const rowData: string[] = isAnchorRowData(row) ? row.rowData : row;
+
+    let onClick: undefined | ((evt: MouseEvent<HTMLTableRowElement>) => void) = undefined;
+    let rowModifierClasses = "";
+
+    if (isAnchorRowData(row)) {
+        onClick = () => {
+            row.onRowClick && row.onRowClick(row.href);
```

Comment:
Should this default to some function (like `document.location.assign(row.href)`)? I feel like that would be the expected behavior, and if you don't pass a function—like in the example—nothing happens when you click the row outside the first data cell.

Reply:
Yes you are absolutely are right and we should navigate by default.

Reply:
I also added `preventDefault` on the anchor element so that all navigation goes through the TableRow click handler.